doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,656 @@
+
+nrfx
+----
+
+See the `Changelog for nrfx 2.3.0`_ for detailed information.
+
+MCUboot
+=======
+
+* Updated MCUboot to facilitate using it as the second stage bootloader:
+
+  * Added minimal configuration overlay file for MCUboot that makes it fit within 16 kB when MCUboot is used as the second stage bootloader.
+    Updated :ref:`documentation <ug_bootloader_adding>` with specific instructions on how and when to use this configuration.
+
+sdk-mcuboot
+-----------
+
+The `sdk-mcuboot`_ fork in |NCS| contains all commits from the upstream MCUboot repository up to and including ``5a6e18148d``, plus some |NCS| specific additions.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+  * Fixed an issue where after erasing an image, an image trailer might be left behind.
+  * Added a ``CONFIG_BOOT_INTR_VEC_RELOC`` option to relocate interrupts to application.
+  * Fixed single image compilation with serial recovery.
+  * Added support for single-image applications (see ``CONFIG_SINGLE_IMAGE_DFU``).
+  * Added a ``CONFIG_BOOT_SIGNATURE_TYPE_NONE`` option to disable the cryptographic check of the image.
+  * Reduced the minimum number of members in SMP packet for serial recovery.
+  * Introduced direct execute-in-place (XIP) mode (see ``CONFIG_BOOT_DIRECT_XIP``).
+  * Fixed kramdown CVE-2020-14001.
+  * Modified the build system to let the application use a private key that is located in the user project configuration directory.
+  * Added support for nRF52840 with ECC keys and CryptoCell.
+  * Allowed to set VTOR to a relay vector before chain-loading the application.
+  * Allowed using a larger primary slot in swap-move.
+    Before, both slots had to be the same size, which imposed an unused sector in the secondary slot.
+  * Fixed bootstrapping in swap-move mode.
+  * Fixed an issue that caused an interrupted swap-move operation to potentially brick the device if the primary image was padded.
+  * Various fixes, enhancements, and changes needed to work with the latest Zephyr version.
+
+Build system
+============
+
+* Updated :ref:`partition_manager`:
+
+  * Added RAM partitioning through the partition manager.
+  * Added the :c:func:`ncs_add_partition_manager_config` function that allows out-of-tree users to specify partition manager configuration files.
+  * Added a warning if no static partition manager configuration is provided when one image (or more) is not built from source in a multi-image build.
+
+* Enabled choosing a build strategy for Zephyr's :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample when it is built as a child image.
+  See :ref:`ug_multi_image` for details.
+* Improved multi-core builds by disassociating domain names from board names.
+* Bugfixes:
+
+  * Fixed a bug where :file:`merged_domains.hex` would be generated in single domain builds.
+  * Fixed a bug where :file:`zephyr/merged.hex` would not be updated when rebuilding a project.
+
+Zephyr
+======
+
+* Added:
```

Comment:
This section ("Added") should be removed, we refer to the 2.4.0 release notes now, so no need to have a separate section here.